package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.22.1",
+    "version": "0.22.2",
```

Comment:
@psivakumaran you can remove this. I have taken care of version changes in PR https://github.com/EMCECS/clarity-react/pull/93. I will create tar package for both changes once your PR gets merged.